### ISSUE TITLE
overview figure

### ISSUE THREAD

**Issue by waehlisch** [anr-bmbf-pivot/draft-dns-over-coaps#7]
the introduction should include an overview figure that presents all important components. the current state makes it surprisingly hard to follow. for example, reading the Introduction and then Selection of DoC Server, i doubt that the reader easily understands what a URI template is and why it is needed. a decent figure could easily close this gap. (i know that RFC 8484 is not a good poster child here.)

**Comment by miri64** [comment]
See d1a8674.
